### PR TITLE
Add user save failed warning

### PR DESCRIPTION
**Pull request by henrywang1**
Motivated by the comments in #91, I add code to display error message when `@user.save` failed.

### CODE REVIEW

File: zlp-scheduler/app/controllers/users_controller.rb [hunk 1]
```diff
@@ -46,6 +46,18 @@ def delete_user
...
+  def create_temp_student(row)
+    user = User.new
+    user.firstname = row[0]
+    user.lastname = row[1]
+    user.uin = row[2]
+    user.email = row[3]
+    user.role = 'student'
+    user.password = 'Temp'
+    user.activate = false
+    user
```
Comment (BWitemeyer) [inline zlp-scheduler/app/controllers/users_controller.rb:58]: Does this line actually do anything?

Comment (henrywang1) [reply zlp-scheduler/app/controllers/users_controller.rb]: I have updated it from `user` to `return user` to make it more explicit.

tl;dr,
- Remove line 58, the function returns `nil`
- Write `user` in line 58, the function returns the local variable user
- Write `return user`, same as above
